guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,465 @@
+# Deploying Akka application on Kubernetes
+
+## The problem
+
+You've created an application using [Akka](http://akka.io/) with [Akka Clustering](http://doc.akka.io/docs/akka/current/scala/common/cluster.html) enabled. After evaluating all of your deployment options, you've chosen to deploy to [Kubernetes](https://kubernetes.io/) to leverage the facilities it provides for automated deployment, scaling, and management of containerized applications. This raises a problem though: how do you deploy an Akka application on Kubernetes? If Akka cluster is required by the application, this requires one or more seed node to be specified to bootstrap the cluster. How do we setup Akka cluster when deploying to Kubernetes? Finally, running an application on Kubernetes requires containerization. How exactly do we deploy these systems to Kubernetes?
+
+## The solution
+
+This guide will cover the steps required to deploy an Akka based application to Kubernetes.
+
+_If your application is based on Lagom or Play, refer to [Deploying Microservices to Kubernetes](http://todo-linkfor information on deploying it to Kubernetes, including how to configure the Cassandra persistence and service discovery between Lagom apps. It is worth noting that the Akka cluster setup for Lagom based applications follows the same steps outlined by this guide._
```

Comment:
Reworded this, so instead of "how to configure Cassandra persistence" now it's "how to deploy Cassandra for the purpose of Akka Persistence"